### ISSUE TITLE
question: Can we extend the list of allowed licenses in `deny.toml` with "Apache-2.0 WITH LLVM-exception"?

### ISSUE THREAD

**Issue by HenningHolmDE** [duesee/imap-codec#514]
Adding PyO3 as a dependency for creating Python bindings (#359) against `imap-codec` will bring in additional dependencies. One of these dependencies is `target-lexicon` which is released under the "Apache-2.0 WITH LLVM-exception" license which is currently not allowed in `deny.toml`:
```
error[rejected]: failed to satisfy license requirements
  ┌─ registry+https://github.com/rust-lang/crates.io-index#target-lexicon@0.12.14:4:12
  │
4 │ license = "Apache-2.0 WITH LLVM-exception"
  │            ^^^^^^^^^^--------------------
  │            │
  │            license expression retrieved via Cargo.toml `license`
  │            rejected: not explicitly allowed
  │
  = Apache-2.0 - Apache License 2.0:
  =   - OSI approved
  =   - FSF Free/Libre
  = target-lexicon v0.12.14
    └── (build) pyo3-build-config v0.21.2
        ├── (build) pyo3 v0.21.2
        │   └── imap-codec-python v2.0.0-alpha.1
        ├── (build) pyo3-ffi v0.21.2
        │   └── pyo3 v0.21.2 (*)
        └── pyo3-macros-backend v0.21.2
            └── pyo3-macros v0.21.2
                └── pyo3 v0.21.2 (*)
```

Can we just add the license to the list of allowed licenses in `deny.toml` or do you see any problem with this?

**Comment by duesee** [comment]
I don't see any (obvious) problem. Sounds similar enough to me that I would just add it w/o being totally sure about the consequences. Let's add it :-)

**Comment by HenningHolmDE** [comment]
Okay, thanks for the quick reply. Adding the license to the allow list is now part of #513

**Comment by duesee** [comment]
Closed by #513
